integreat_cms/cms/forms/organizations/organization_form.py

```diff
@@ -26,3 +34,51 @@ class Meta:
         widgets = {
             "icon": IconWidget(),
         }
+
+    def __init__(self, **kwargs):
+        r"""
+        Initialize organization form
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+        """
+        super().__init__(**kwargs)
+        if self.instance:
+            # logger.info(Organization.users.all())
+            logger.info(self.instance.slug)
+            logger.info(self.instance.pages.all())
+            # logger.info(self.instance.users.all())
```

Comment:
```suggestion
```
The instance is already logged [here](https://github.com/digitalfabrik/integreat-cms/blob/5c3265e28770a5545ddd9dc881ecfef41b8e6c25/integreat_cms/cms/forms/custom_model_form.py#L47-L61).

Reply:
your right :D